backend/src/main/java/org/launchcode/fancyrats/models/Job.java

```diff
@@ -42,6 +42,26 @@ public class Job {
     @NotNull(message = "Description required")
     private String description;
 
+    public static enum animalType{
```

Comment:
Is there a reason you defined the enum in the Job class? Also I don't see animalType getting used anywhere